wgsl/index.bs

```diff
@@ -6857,6 +6853,20 @@ That's not a full user-defined function declaration.
     [=Component-wise=] when |T| is a vector.
     (GLSLstd450Fma)
 
+  <tr algorithm="fmod">
+    <td>|T| is [FLOATING]
+    <td class="nowrap">`fmod(`|e1|`:` |T| `, `|e2|`:` |T| `) -> ` |T|
+    <td>Returns the floating point remainder of |e1| divided by |e2|, where sign of non-zero result matches sign of |e2|.
+    [=Component-wise=] when |T| is a vector.
+    (OpFMod)
+
+  <tr algorithm="frem">
+    <td>|T| is [FLOATING]
+    <td class="nowrap">`frem(`|e1|`:` |T| `, `|e2|`:` |T| `) -> ` |T|
+    <td>Returns the floating point remainder of |e1| divided by |e2|, where sign of non-zero result matches sign of |e1|.
```

Comment:
When the sign matches |e1|, it's the std::fmod function from C++.  https://en.cppreference.com/w/cpp/numeric/math/fmod 



That's the unfortunate thing we found with the SPIR-V instructions: C++ std::fmod is OpFRem.

So I'd swap the WGSL names  `frem` and `fmod`.


<google-sheets-html-origin><style type="text/css"><!--td {border: 1px solid #ccc;}br {mso-data-placement:same-cell;}--></style>

x | y | floor(x/y) | x-y*(floor(x/y)) | OpFMod | trunc(x/y) | x-y*trunc(x/y)<br>std::fmod | OpFRem
-- | -- | -- | -- | -- | -- | -- | --
15 | 4 | 3 | 3 | 3 | 3 | 3 | 3
15 | -4 | -4 | -1 | -1 | -3 | 3 | 3
-15 | 4 | -4 | 1 | 1 | -3 | -3 | -3
-15 | -4 | 3 | -3 | -3 | 3 | -3 | -3

Reply:
Done